charts/kspm-collector/Chart.yaml

```
@@ -2,8 +2,8 @@ apiVersion: v2
 name: kspm-collector
 description: Sysdig KSPM collector
 
-version: 0.11.0
-appVersion: 1.37.0
+version: 0.12.0
```

I see that every other chart receive a patch bump. Shouldn't it be
```suggestion
version: 0.11.1
```
?
In that case please update also the dependency in `sysdig-deploy` accordingly